dyn_em/module_em.F

```diff
@@ -333,7 +337,14 @@ SUBROUTINE rk_tendency ( config_flags, rk_step,                           &
    REAL    :: kdift, khdq, kvdq, cfn, cfn1, cf1, cf2, cf3
    INTEGER :: i,j,k
    INTEGER :: time_step
+   INTEGER :: rk_order
+   REAL    :: dt_step
 
+! IEVA declarations
+   REAL , DIMENSION( ims:ime, kms:kme, jms:jme ) :: wwE, wwI  
```

Comment:
SHould these two be tile sized? Do they go into other routines as memory sized? For OpenMP, this is both a memory concern (every thread allocates the entire MPI patch), and a possible place for a race condition (multiple OpenMP threads have the identical (i,j) in memory, and whoever writes it out last WINS.

Reply:
To me, it looks like these ww explicit and implicit ARE passed into existing routines, so the memory dims are reasonable. Would it be better to have these two variables declared in the Registry? They could be packaged with the IEVA routine. Since this is ARW, there would be no impact on the NMM registry.

Reply:
Yea, the memory dims are reasonable, IMHO, but I also know that WRF is not necessarily supposed to use these as temps.  Because I have to assess the halo for wwE and wwI for horizontal averaging, I thought that was a reasonable reason for violating that rule.

I dont see a big advantage for moving this to the Registry - would they then have to be passed in explicitly to the rk_tendency and rk_scalar_tend routines?  Or is there another way?  Is there a current example of doing this?

Reply:
Lou,
This seems fine since these variables are being passed into routines that expect memory dims

Reply:
@davegill @louiswicker Based on things I learned recently, these two 3D arrays as well as the two 2D arrays (mut_old, mut_new) are likely need to be declared tile dimensions. If you go one level deeper, the calculations are done using tile dimensions anyway. Please review.